### PR TITLE
start to translate chapters requested.

### PR DESCRIPTION
**Pull request by Alex1989Wang**
看到可能还有五个章节没有在Contributing.md上认领，自己认领了这5个章节。
刚刚完成了一个。

### CODE REVIEW

**Comment by wsdjeg** [comment]
谢谢！

File: chapter/List-Of-Plugins.md [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
 
-#### [以功能区分](#by-topic-1)
+#### [以功能区分](#以功能区分-1)
 
 - [文本对齐](#文本对齐)
-- [Building and linting](#building-and-linting)
-- [Code completion](#code-completion)
+- [构建和可疑代码标记](#构建和可疑代码标记)
+- [代码补全](#代码补全)
 - [Cycle](#cycle)
```
Comment (Alex1989Wang) [inline chapter/List-Of-Plugins.md]: Cycle一词不知怎么翻译才好

File: chapter/List-Of-Plugins.md [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
-- [Text objects](#text-objects)
+- [注释工具](#注释工具)
+- [分割符](#分割符)
+- [模糊搜索](#模糊搜索)
+- [Grep 工具](#grep-工具)
+- [缩进](#缩进)
+- [文件导航](#文件导航)
+- [代码片段](#代码片段)
+- [状态栏](#状态栏)
+- [文字环绕符](#文字环绕符)
```
Comment (Alex1989Wang) [inline chapter/List-Of-Plugins.md]: 原文是Surround一词；
我勉强翻译为‘文字环绕符’

Comment (wsdjeg) [inline chapter/List-Of-Plugins.md]: 这个翻译挺好

File: chapter/List-Of-Plugins.md (hunk 1 above)
Comment (wsdjeg) [inline chapter/List-Of-Plugins.md]: @mhinz, I am not sure if this word can be translate as `toggle` ( 切换 ）？

Comment (mhinz) [inline chapter/List-Of-Plugins.md]: Hmm, not really. Toggling means switching between only 2 states (binary, on/off), whereas "cycling" means switching between multiple states, e.g. buffers.

**Pull request merged**